### DISCUSSION TITLE
Gestione Magazzino

### DISCUSSION THREAD

**Discussion by davidefiumara** [fattureincloud/api#329] [Q&A] [answered]
Salve, è possibile aggiornare il magazzino si FattureInCloud da woocommerce? Si possono esportare gli ordini da woocommerce a FattureInCloud?

**Comment by emajo** [comment, marked as answer]
Ciao @davidefiumara, può trovare delle integrazioni sviluppate da sviluppatori terzi per woocommerce sul nostro [App Store](https://apps.fattureincloud.it/apps), alternativamente può creare la sua integrazione utilizzando le nostre API pubbliche, [qui](https://developers.fattureincloud.it) trova tutta la documentazione necessaria.

Rimaniamo a disposizione,
Emanuele Saccomandi.

**Reply by valmoz** [reply]
Buongiorno @davidefiumara ha altre domande a riguardo?

Grazie
Mauro

**Comment by davidefiumara** [comment]
Buongiorno,per il momento no, eventualmente la disturbo nuovamente.Grazie.
    Il venerdì 30 giugno 2023 alle ore 11:29:43 CEST, Mauro Valota ***@***.***> ha scritto:  
 
 


Buongiorno @davidefiumara ha altre domande a riguardo?

Grazie
Mauro

**Reply by valmoz** [reply]
Perfetto, buona giornata.
